### PR TITLE
ANC logic - fix 1

### PR DESCRIPTION
**Pull request by anvmn**
#123 
#124 
#203 
#210

### CODE REVIEW

**Comment by anvmn** [comment]
Demo for #210:

Peek 2022-04-07 12-03

**Comment by anvmn** [comment]
Selection_641

**Comment by anvmn** [comment]
Peek 2022-04-19 22-29

**Comment by anvmn** [comment]
@nedSaf please review

File: client/src/elm/Pages/Prenatal/Activity/View.elm [hunk 1]
```diff
@@ -1708,24 +1728,36 @@ viewNextStepsContent language currentDate isChw assembled data =
...
-                            [ button
-                                [ classList [ ( "ui fluid primary button", True ), ( "disabled", tasksCompleted /= totalTasks ) ]
-                                , onClick saveMsg
-                                ]
-                                [ text <| translate language Translate.Save ]
-                            ]
+                        case task of
+                            NextStepsWait ->
+                                viewPauseEncounterButton language
+                                    -- Button is enabled becausethere are
```
Comment (nedSaf) [inline client/src/elm/Pages/Prenatal/Activity/View.elm]: => `because there`

File: client/src/elm/Pages/Prenatal/Activity/View.elm [hunk 2]
```diff
@@ -1708,24 +1728,36 @@ viewNextStepsContent language currentDate isChw assembled data =
...
+                                    -- no actual tasks to be performed.
+                                    True
+                                    -- When saving, we'll also 'pause' the encounter
+                                    -- which actualy navigates to main menu page.
+                                    -- The encountre is closed on second phase.
```
Comment (nedSaf) [inline client/src/elm/Pages/Prenatal/Activity/View.elm]: => `encounter`

**Review by nedSaf — APPROVED** [review]
looks good, two little typos and you can MWG :)

**Pull request merged**